lib/table.js

```diff
@@ -58,7 +58,16 @@ Table.prototype.update = function(conditions, fields, next) {
     options.single = true;
   }
 
-  if(_.isObject(fields) === false) throw "Update requires a hash of fields=>values to update to";
+  assert(_.isObject(fields), "Update requires a hash of fields=>values to update to");
```

Comment:
Also using `assert` here to be consistent with other methods. Was previously throwing a string.